lib/bloc/authentication/authentication_bloc.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:bloc/bloc.dart';
+
+import 'authentication_event.dart';
+import 'authentication_state.dart';
+import '../../services/authentication_services.dart';
+
+class AuthenticationBloc
+    extends Bloc<AuthenticationEvent, AuthenticationState> {
+  final AuthenticationService _authenticationService;
+
```

Comment:
ok. i dont think we need a seperate authentication bloc. this is over compliecation.

we can just add another event "CHECK_AUTHENTICATED" in the login block itself. 

as this is just to check.

Reply:
so we can remove this whole bloc